tests/Behat/Context/UI/ManagingAdministratorsContext.php

```diff
@@ -25,16 +31,54 @@ final class ManagingAdministratorsContext implements Context
     /** @var UserRepositoryInterface */
     private $adminUserRepository;
 
+    /** @var ObjectManager */
+    private $administratorManager;
+
+    /** @var ExampleFactoryInterface */
+    private $adminUserExampleFactory;
+
+    /** @var SharedStorageInterface */
+    private $sharedStorage;
+
     public function __construct(
         AdministrationRolesElementInterface $administrationRolesElement,
         AdminUserIndexPageInterface $indexPage,
         UpdatePageInterface $updatePage,
-        UserRepositoryInterface $adminUserRepository
+        UserRepositoryInterface $adminUserRepository,
+        ObjectManager $administratorManager,
+        ExampleFactoryInterface $adminUserExampleFactory,
+        SharedStorageInterface $sharedStorage
     ) {
         $this->administrationRolesElement = $administrationRolesElement;
         $this->indexPage = $indexPage;
         $this->updatePage = $updatePage;
         $this->adminUserRepository = $adminUserRepository;
+        $this->administratorManager = $administratorManager;
+        $this->adminUserExampleFactory = $adminUserExampleFactory;
+        $this->sharedStorage = $sharedStorage;
+    }
+
+    /**
+     * @When I add a new administrator :email named :firstName with :lastname last name and :administrationRole administration role
```

Comment:
It is not a Ui step, it is rather a domain one. You should probably move this step to other context in `Domain` namespace